docs/aws-ec2/README.md

```diff
@@ -0,0 +1,22 @@
+# EC2 Developer Setup
+
+## Background
+
+For easy development purposes, it is beneficial to setup a ec2 instance for development as the memory requirements for armada can be quite intense.  For example, a mac M1 with 8 gb is too small for development.  Some images will also not be supported on a M1 so it is beneficial to setup a ec2 instance for development.
+
+## Instructions
+
+- We suggest a t3.xlarge instance from aws ec2 with Amazon Linux.  16 GB of memory is suggested.
```

Comment:
Also mention that the user should select AmazonLinux (as opposed to debian etc)

Reply:
I have found that 8 GB of memory wasn't enough.  The t2.large only has 8 gb of memory.

@dejanzele suggested me the t3.large because that was what we are using for our EKS cluster.

Reply:
Ah OK